### DISCUSSION TITLE
Weird behavior with mini.jump2d

### DISCUSSION THREAD

**Discussion by 22ispencer** [m4xshen/hardtime.nvim#102] [Q&A]
How can I disable hardtime from reading input when i enter mini.jump2d, when i try to use it by typing `<Enter>jb` or similar it doesn't do the jumping motion until i repeat the jump keys, everything was working before i installed hardtime, I'm trying to break my habit of holding j to go down.

**Comment by makew0rld** [comment]
I filed an issue for this: https://github.com/m4xshen/hardtime.nvim/issues/127

**Comment by m4xshen** [comment]
@22ispencer you can now update hardtime.nvim to latest version. mini.jump2d should work fine now!
